test/functional/cmdLineTests/jython/jython.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2021, 2021 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<suite id="jython" timeout="300">
+    
+    <test id="jython-hello">
+        <command>$EXE$ -XshowSettings:vm -Dpython.options.showJavaExceptions=true -Dpython.options.includeJavaStackInExceptions=true -Dpython.options.showPythonProxyExceptions=true -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=n -cp $Q$$JARPATH$$Q$:.  JythonHello</command>
```

Comment:
Instead of hard coding the command line, can we run with a number of different modes?
`-Xdebug -Xnoagent` do nothing.
`-Djava.compiler=NONE` is the same as `-Xint`
There are modes that enable debug as per `-Xrunjdwp:transport=dt_socket,address=8787,server=y,suspend=n` such as Mode108.

We can run with NoOptions
Modes:
```
108: -Xgcpolicy:gencon -Xdebug -Xrunjdwp:transport=dt_socket,address=8888,server=y,onthrow=no.pkg.foo,launch=echo -Xint -Xnocompressedrefs 
109: -Xgcpolicy:gencon -Xint -Xnocompressedrefs 
116: -Xgcpolicy:gencon -Xdebug -Xrunjdwp:transport=dt_socket,address=8888,server=y,onthrow=no.pkg.foo,launch=echo -Xjit:count=0 -Xnocompressedrefs 
608: 	
-Xcompressedrefs -Xgcpolicy:gencon -Xdebug -Xrunjdwp:transport=dt_socket,address=8888,server=y,onthrow=no.pkg.foo,launch=echo -Xint 
609: -Xcompressedrefs -Xgcpolicy:gencon -Xint 
616: Not sure it exists, but we could add it  -Xgcpolicy:gencon -Xdebug -Xrunjdwp:transport=dt_socket,address=8888,server=y,onthrow=no.pkg.foo,launch=echo -Xjit:count=0 -Xcompressedrefs 
```

Reply:
@OscarQQ I do not think we have 616. To add 616, please update into https://github.com/adoptium/TKG/blob/master/resources/modes.xml and ottawa.csv